integrations/pagerduty/pagerduty-incidents/README.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The PagerDuty integration is a Sensu Event Pipeline that generates alerts in the PagerDuty incident response service.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* `pagerduty` [pipeline]
+* `pagerduty` [handler]
+* `pagerduty_integration_key` [secret]
+* `sensu/sensu-pagerduty-handler` [asset]
```

Comment:
Do we want to have the versioned asset in the Readme resources list?

Reply:
I could go either way on this; omitting the version just means there's less maintenance when we publish a new asset, but being specific _is_ a good thing.